src/Cms/FileVersion.php

```diff
@@ -132,7 +133,7 @@ protected function setOriginal($original)
      */
     public function toArray(): array
     {
-        $array = array_merge($this->parentToArray(), [
+        $array = array_merge($this->asset()->ToArray(), [
```

Comment:
Are you sure this shouldn't be `$this->asset()->toArray()` (with a small `t`)?

Reply:
Thanks for the catch